### PR TITLE
Feature/stij 50: Fixed label atom + form previews

### PR DESCRIPTION
**Pull request by meirege**
## Description
Did a lot of fixes to the form previews. For attributes should work for their respective form elements and it is now easier to re-use these form components inside Fractal.

### CODE REVIEW

File: components/21-atoms/inputs/input-checkbox/input-checkbox.twig [hunk 1]
```diff
@@ -1 +1,5 @@
-<input type="checkbox" id="{{ id }}" name="checkbox" class="checkbox {{ modifier }}" /><label for="{{ id }}">{{ label }}</label>
+<input type="checkbox"
+       id="{{ checkbox_id}}"
```
Comment (Jeroen005) [inline components/21-atoms/inputs/input-checkbox/input-checkbox.twig]: space

File: components/31-molecules/fields/checkbox/checkbox.config.json [hunk 2]
```diff
@@ -5,8 +5,26 @@
...
-    "field_description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec laoreet, urna sit amet convallis rhoncus, felis ex pellentesque neque, nec ultrices dui enim ut diam. Nam pellentesque velit pharetra, accumsan ante at, gravida turpis. Cras venenatis velit ut ipsum molestie pretium. Vivamus tellus metus, pretium eu nibh vitae, eleifend luctus dolor. Morbi feugiat tristique ante eu egestas. Nunc ligula magna, fringilla sed consequat id, pellentesque at risus. Nunc ex elit, tincidunt at lorem vel, mattis gravida nisl. Praesent sodales nibh massa, id ultrices metus viverra id."
+    "description": "Description checkboxes.",
+
+    "label_checkbox1": "Checkbox option 1",
+    "checkbox1_name": "checkbox-1",
+    "checkbox1_id": "checkbox-1",
+
+    "checkbox1_label": "Label checkbox 1",
+    "checkbox1_label_for": "checkbox-1",
+    "checkbox1_label_optional": "Optional",
```
Comment (Jeroen005) [inline components/31-molecules/fields/checkbox/checkbox.config.json]: is the optional label here correct? Don't know if it should be possible to place this kind of label next to the option itself.

File: components/31-molecules/fields/radio/radio.twig [hunk 3]
```diff
@@ -1,16 +1,47 @@
...
+
+      {% include '@input-radio' with {
+        'label': label_radio1,
+        'radio_name': radio1_name,
+        'radio_id': radio1_id
+      } %}
+      {% include '@label' with {
+        "label": radio1_label,
+        "for": radio1_label_for,
+        "label_optional": radio1_label_optional
```
Comment (Jeroen005) [inline components/31-molecules/fields/radio/radio.twig]: same here with optional

File: components/31-molecules/fields/checkbox/checkbox.config.json (hunk 2 above)
Comment (meirege) [reply components/31-molecules/fields/checkbox/checkbox.config.json]: Fixed.

File: components/31-molecules/fields/radio/radio.twig (hunk 3 above)
Comment (meirege) [reply components/31-molecules/fields/radio/radio.twig]: Fixed.

**Pull request merged**